community/xmlsec1/depends

```diff
@@ -0,0 +1,2 @@
+libressl
```

Comment:
I'm not sure about dependencies here.
According to FAQ https://www.aleksey.com/xmlsec/faq.html xmlsec can be built against
different crypto libraries (openssl/libressl, gnutls, libgcrypt, nss).
I only added `libressl` to depends file to satisfy minimum requirements, but would it make more sense to include `gnutls`, `libgcrypt` and `nss` here as well?

Reply:
It would not. You should be able to use configure flags to limit it to libressl, if not it is fine as-is. :+1:

Reply:
Added flags and a missing `libtool` dependency. Should be good now